pkg/resolver/search.go

```diff
@@ -73,7 +75,9 @@ func (s *SearchResult) Related() []SearchRelatedResult {
 	var r []SearchRelatedResult
 
 	if len(s.uids) > 0 {
+		startTime := time.Now()
 		r = s.getRelations()
+		klog.Info("Time taken to resolve relationships: ", time.Since(startTime))
```

Comment:
**Maybe this could be a separate PR.** Can we use the same pattern from the indexer?
- Log if it takes more than the expected time.
- Log if v=4